src/neptune_tensorboard/integration/__init__.py

```diff
@@ -47,9 +50,30 @@ def integrate_with_tensorflow(run, base_namespace):
             raise Exception(message.format(version))
 
 
+if IS_PYT_AVAILABLE:
+    import torch
+
+    from neptune_tensorboard.integration.pytorch_integration import patch_pytorch
+
+    def integrate_with_pytorch(run, base_namespace):
+        version = "<unknown>"
+        try:
+            # noinspection PyUnresolvedReferences
+            version = parse_version(torch.__version__)
+
+            if version >= parse_version("1.9.0"):
+                patch_pytorch(run, base_namespace)
+        except AttributeError:
+            message = "Unrecognized PyTorch version: {}. Please make sure " "that the PyTorch version is >=1.9.0"
+            raise Exception(message.format(version))
+
+
 def enable_tensorboard_logging(run, *, base_namespace="tensorboard"):
     if IS_TF_AVAILABLE:
         integrate_with_tensorflow(run, base_namespace)
-    else:
-        msg = "neptune-tensorboard: Tensorflow was not found, please ensure that it is available."
+    if IS_PYT_AVAILABLE:
+        integrate_with_pytorch(run, base_namespace)
+
+    if not (IS_PYT_AVAILABLE or IS_TF_AVAILABLE):
```

Comment:
`else` does the same thing to my knowledge.

Reply:
I don't think else will capture `IS_TF_AVAILABLE`.

We want to patch each library if available so each `IS_*_AVAILABLE` has to be its own check and with that I don't think else is possible. Wdyt?

Reply:
Let's ask the Great One.
![Screenshot 2023-06-12 at 23 46 06](https://github.com/neptune-ai/neptune-tensorboard/assets/917619/0f5d7e22-19c8-43a5-b429-35b53a5396db)

Reply:
Ohh, I see now. It's double if (non-related to each other). I thought it was if ... elif . Sorry, my fault.